File: C++/1165.cpp
#include<iostream>

using namespace std;

int main(){
    int casosDeTeste;
    int entrada, contador;

    cin >> casosDeTeste;

    for(int i = 1; i <= casosDeTeste; i++){
        cin >> entrada;
        contador = 0;

        for(int j = 1; j <= entrada; j++){
            if(entrada % j == 0){
                contador++;
            }
        }
        if(contador == 2){
            cout << entrada << " eh primo" << endl;
        }else{
            cout << entrada << " nao eh primo" << endl;
        }
    }
    return 0;
}

File: JavaScript/1165.js
var input = require('fs').readFileSync('stdin', 'utf8');

var lines = input.split('\n');
/*Começa aqui */
let casosTeste;
let numero, contador;

casosTeste = parseInt(lines[0])


for(let i = 1; i <= casosTeste; i++){
    numero = parseInt(lines[i]);
    contador = 0;

    for(let j = 1; j <= numero; j++){
        if(numero % j === 0){
            contador++;
        }
    }
    if(contador === 2){
        console.log(`${numero} eh primo`)
    }else{
        console.log(`${numero} nao eh primo`)
    }
}